src/lando/main/models/repo.py

```diff
@@ -120,6 +118,8 @@ def get_scm(self) -> AbstractSCM:
         if not self._scm:
             if self.scm == SCM_HG:
                 self._scm = HgSCM(self.path)
+            elif self.scm == SCM_GIT:
+                self._scm = GitSCM(self.path, self.default_branch)
```

Comment:
Left some comments in the other PR (#154) that should impact this implementation

Reply:
For reference: https://github.com/mozilla-conduit/lando/pull/154#discussion_r1868275696

Let's talk about the best way to do this when we catch up.